### PR TITLE
[KOGITO-5299] - Update Examples following new Addons structure

### PR DESCRIPTION
**Pull request by ricardozanini**
Many thanks for submitting your Pull Request :heart:! 

See: https://github.com/kiegroup/kogito-runtimes/pull/1367

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Comment by ricardozanini** [comment]
Jenkins retest this

File: dmn-drools-quarkus-metrics/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Import the following dependency in your `pom.xml`:
 ```XML
 <dependency>
  <groupId>org.kie.kogito</groupId>
```
Comment (cristianonicolai) [inline dmn-drools-quarkus-metrics/README.md:21]: @ricardozanini wdyt about having the groupid to identify the addons.
like: `org.kie.kogito.addons.quarkus` and `org.kie.kogito.addons.springboot` then we could simplify the artifactId ?

Comment (MarianMacik) [reply dmn-drools-quarkus-metrics/README.md:21]: +1, we don't need to have everything under `org.kie.kogito` I think. The project is fairly big now, so we can have longer groupIds.

**Review by MarianMacik — APPROVED** [review]

File: dmn-drools-quarkus-metrics/README.md (hunk 1 above)
Comment (ricardozanini) [reply dmn-drools-quarkus-metrics/README.md:21]: Quarkus is bigger and everything is under the same group :thinking: (as far as I know :rofl:) . @MarianMacik please see this thread: https://kie.zulipchat.com/#narrow/stream/232676-kogito/topic/The.20Addons.20Restructure.20Thread

**Review by r00ta — APPROVED** [review]
Hi @ricardozanini, do you already have a PR for the kie docs as well?

**Comment by ricardozanini** [comment]
> Hi @ricardozanini, do you already have a PR for the kie docs as well?

Not yet, I'm working on it.

**Pull request merged**